lib/cli/src/Cardano/CLI.hs

```diff
@@ -594,6 +595,25 @@ cmdTransactionList = command "list" $ info (helper <*> cmd) $ mempty
             mTimeRangeEnd
             (ApiT <$> mOrder)
 
+-- | Arguments for 'transaction submit' command
+data TransactionSubmitArgs = TransactionSubmitArgs
+    { _port :: Port "Wallet"
+    , _payload :: PostExternalTransactionData
```

Comment:
I suppose this should really be `ByteString` (as discussed on Slack)

Reply:
I use : 
```
newtype PostExternalTransactionData = PostExternalTransactionData (payload :: ByteString}
```
in order to deliver a number of instances and not use bytestring instances